ui/panels-plugin/src/plugins/time-series-chart/TimeSeriesChartPanel.tsx

```diff
@@ -46,6 +46,15 @@ export function TimeSeriesChartPanel(props: TimeSeriesChartProps) {
     spec: { queries, thresholds, y_axis },
     contentDimensions,
   } = props;
+  const theme = useTheme();
+
+  const contentPadding = parseInt(theme.spacing(1.5), 10);
+  const adjustedContentDimensions: typeof contentDimensions = contentDimensions
+    ? {
+        width: contentDimensions.width - contentPadding * 2,
+        height: contentDimensions.height - contentPadding * 2,
+      }
+    : undefined;
```

Comment:
This feels a little yucky, and the time series chart could probably be significantly refactored to do something different. However, I was inclined not to shave that yak as part of this PR and leave it for the next time we're putting more significant attention into the time series panel and need to refactor anyways (e.g. adjusting the legend to be more in line with the mocks from design).